test/pieces-pages.js

```diff
@@ -50,6 +50,15 @@ describe('Pieces Pages', function() {
         }
       },
       afterInit: function(callback) {
+        console.log('>> afterInit');
```

Comment:
Do you want to keep this log output around?

Reply:
Oops, fixed.